### DISCUSSION TITLE
Zapier Bad Request

### DISCUSSION THREAD

**Discussion by agencydoctor** [deepgram/community#621] [General help]
### Which Deepgram product are you using?

Deepgram API

### Details

The API was functioning properly, but now it's returning a "Bad Request" error in Zapier.

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by SandraRodgers** [comment]
Hi @agencydoctor, could you tell me a little about the Zapier workflow? What is the trigger and which Deepgram action are you using? Thanks.
